site/class-2-style/niyorn/index.js

```diff
@@ -0,0 +1,36 @@
+function render(b, c) {

```

Comment:
You should really rename your one letter variables!

This doesn’t make a bunch of sense, but 🤷‍♂️ I’ll merge.